activation_viewer/loader/styles.py

```diff
@@ -3,23 +3,60 @@ def getSld(geom_type):
     return styles[geom_type]
 
 styles = {
-'l':
+'Grading_l':
```

Comment:
@maririv thanks for this. As those are just keys of a dict, could you please keep them lowercase?